index.js

```diff
@@ -947,9 +957,9 @@
   //# and :: a -> a -> a
   //.
   //. Takes two values of the same type and returns the second value
-  //. if the first is "true"; the first value otherwise. An array is
-  //. considered "true" if its length is greater than zero. The Boolean
-  //. value `true` is also considered "true". Other types must provide
+  //. if the first is “true”; the first value otherwise. An array is
+  //. considered “true” if its length is greater than zero. The Boolean
+  //. value `true` is also considered “true”. Other types must provide
```

Comment:
I wanted to use curly quotes around “plucking”, and wanted to be consistent. I'll switch to curly quotes in a separate pull request then rebase this branch. :)